projects/demo/src/modules/experimental/pin/pin.module.ts

```diff
@@ -0,0 +1,31 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {TuiAddonDocModule, tuiGetDocModules} from '@taiga-ui/addon-doc';
+import {TuiNotificationModule, TuiSvgModule} from '@taiga-ui/core';
+import {TuiAppearanceModule, TuiIconModule, TuiPinModule} from '@taiga-ui/experimental';
+
+import {TuiPinExample1} from './examples/1';
+import {TuiPinExample2} from './examples/2';
+import {TuiPinExample3} from './examples/3';
+import {ExampleTuiPinComponent} from './pin.component';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        TuiSvgModule,
+        TuiPinModule,
+        TuiAddonDocModule,
```

Comment:
`tuiGetDocModules` (which is imported in this module) already includes `TuiAddonDocModule`